course_contents/1_intro/notes/07. Membership Operators.md

```diff
@@ -0,0 +1,19 @@
+## IN
+- Returns True if an element exists inside an object
+
+Example:
+```python
+users = ['James', 'Charlie', 'Ana']
+if 'James' in users:
+    print(True)
+```
+
+## NOT IN
+- Returns True if an element does not exist inside an object
+
+Example:
+```python
+users = ['James', 'Charlie', 'Ana']
+if 'Johnny' in users:
+    print(True)
+```
```

Comment:
Don't think we covered if statements yet for this PDF.

Maye just `print("Johnny" in users)` (same for both blocks)?